tfhe/src/integer/client_key/secret_encryption_key.rs

```diff
@@ -0,0 +1,7 @@
+use crate::core_crypto::commons::traits::Container;
+use crate::shortint::client_key::secret_encryption_key::SecretEncryptionKey as ShortintSecretEncryptionKey;
+
+#[derive(Clone)]
+pub struct SecretEncryptionKey<KeyCont: Container<Element = u64>> {
```

Comment:
Do we want generics in the integer API (and even in the shortint one)?

Reply:
Well if we don't we have to manually create types for views, seems rather wasteful don't you think ?

Reply:
Not sure to understand what you mean, but I haven't looked as the usage
If this type is used with more than generic in the code, it's fine to have it generic I guess

Reply:
actually looks like I only use views, so I'll change it to be views only